lib/sparse_matrix.rb

```diff
@@ -16,38 +16,6 @@ def identity(n)
     alias :I :identity
   end
 
-  def rows
```

Comment:
Why are we removing these? They need to be in the interface for the TA to read

Reply:
Moved them back to the classes (had them in the factories)